core/coreapi/coreapi.go

```diff
@@ -15,28 +15,34 @@ type CoreAPI struct {
 	node *core.IpfsNode
 }
 
-// NewCoreAPI creates new instance of IPFS CoreAPI backed by go-ipfs Node
+// NewCoreAPI creates new instance of IPFS CoreAPI backed by go-ipfs Node.
 func NewCoreAPI(n *core.IpfsNode) coreiface.CoreAPI {
 	api := &CoreAPI{n}
 	return api
 }
 
+// Unixfs returns the Unixfs API.
```

Comment:
I feel like there should be a thing after API...
```
// Unixfs returns the Unixfs API *doodad*
```

thoughts @magik6k @lgierth ?

Reply:
how about `interface`?

Reply:
or maybe
`// Unixfs create new instance of IPFS UnixfsAPI backed by go-ipfs Node.`